cmd/loop/liquidity.go

```diff
@@ -324,6 +324,17 @@ var setParamsCommand = cli.Command{
 			Usage: "the confirmation target for loop in on-chain " +
 				"htlcs",
 		},
+		cli.BoolFlag{
+			Name: "easyautoloop",
+			Usage: "set to true to enable easy autoloop, which " +
+				"will automatically dispatch swaps in order to meet " +
+				"the target hot wallet size",
+		},
+		cli.Uint64Flag{
+			Name: "hotwalletsizesat",
```

Comment:
If agree with above, maybe we could call this param `offchainliquiditytargetsats` or just simply `offchaintargetsats`.